instrumentation/opentelemetry-api-beta/src/main/java/io/opentelemetry/auto/instrumentation/opentelemetryapi/vbeta/AbstractInstrumentation.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.opentelemetry.auto.instrumentation.opentelemetryapi;
+package io.opentelemetry.auto.instrumentation.opentelemetryapi.vbeta;
```

Comment:
Is it correct package name, `vbeta`?

Reply:
it's a placeholder for `v1_0`

though our convention is not to have a `vX_Y` package until there are more than one version supported, so i have removed 👍